Modules/IO/TIFF/src/itkTIFFImageIO.cxx

```diff
@@ -456,26 +456,29 @@ TIFFImageIO::ReadImageInformation()
       this->SetPixelType(RGBA);
   }
 
-  if ((this->GetFormat() == TIFFImageIO::PALETTE_GRAYSCALE || this->GetFormat() == TIFFImageIO::PALETTE_RGB) &&
-      m_TotalColors > 0 && this->GetExpandRGBPalette())
```

Comment:
Has the check for `ExpandRGBPalette` been intentionally removed?

Reply:
Yes, because with a non supported palette file and GetExpandRGBPalette to false, we have to reset to the proper RGB component type (line 518)